experimental/pgbouncer_with_vault/install.sh

```diff
@@ -55,12 +55,15 @@ sleep 60
 #=================================================================================================
 # SETUP VAULT
 #-------------------------------------------------------------------------------------------------
-git clone --single-branch --branch v0.1.2 https://github.com/hashicorp/vault-helm.git
+git clone --single-branch --branch master https://github.com/hashicorp/vault-helm.git
+if [ "$SINGLE_NODE_CLUSTER" = true ]; then
+  sed -i '/affinity: |/,+8 s/^/#/' vault-helm/values.yaml
+fi
 sed -i "s/HOST_IP:8500/${CONSUL_NAME}-consul-server:8500/g" vault-helm/values.yaml
 helm install "${VAULT_NAME}" ./vault-helm --set='server.ha.enabled=true'
 rm -rf vault-helm
 
-sleep 15
+sleep 30
```

Comment:
### (not a blocker)
I recommend against using sleep. There is a way to check that the sql database is up by waiting for the service to be ready... 
Something like: 
```shell
kubectl wait --for=condition=ready service/<service-name> --timeout=60s
```
Or some other condition. I am not sure when the service is makred ready, you would have to read bout that, but I think this would be a better approach.

Reply:
ok thanks a lot. I haven;t wanted to use sleep but I didn't know how to do this. So will change it.

Reply:
This is mnostly done but I'm still using sleep while doing the vault initialisation. Will need to do a bit of bash parsing to replace these at a later point but will leave it for now.